### DISCUSSION TITLE
Is ShyFox compatible with the Firefox Gnome Theme? Can it be?

### DISCUSSION THREAD

**Discussion by kbdragon** [Naezr/ShyFox#163] [General]
Hi, I'm currently using the [Firefox Gnome Theme](https://github.com/rafaelmardojai/firefox-gnome-theme) to make it fit the Gnome aesthetic, but I'm super impressed with the features in ShyFox. 

I'd love to combine the two. Is it possible to use both?

**Comment by Naezr** [comment]
No. Everything will probably break when you try to combine it. Both themes are quite complicated (the ShyFox 4.0-alpha is beyond all possible limits).

However, I use GNOME myself, and I think the theme already fits into it pretty well. And I'm already inspired by Adwaita in some aspects, so there may be more GNOME-inspired visual elements in ShyFox in the future. But right now I'm busy learning and getting the super complex ShyFox 4.0 to work well.

**Reply by kbdragon** [reply]
Thanks for the quick response!

I figured that's the case, but no harm in asking.

Is there an easy way to uninstall ShyFox? I'd like to try it out after I remove the Gnome theme.

**Reply by Naezr** [reply]
Just remove or rename the `chrome` folder from your profile. And `user.js` file too

**Reply by kbdragon** [reply]
Brilliant, thanks!
